docs/4-language-usage/7-stored-objects/1-general/g-7170.md

```diff
@@ -0,0 +1,89 @@
+# G-7170: Avoid using an IN OUT parameter as IN or OUT only.
+
+!!! warning "Major"
+    Efficiency, Maintainability
+
+!!! missing "Unsupported in PL/SQL Cop Validators"
+    Rule G-7170 is not expected to be implemented in the static code analysis validators.
```

Comment:
maybe we should state here only the reason. The tag should be explained in the introduction chapter. And here the reason would be something like:

We cannot determine the usage of an `in out` parameter in a reliable way, especially when other units are involved which are maintained in another file.